kiosk/src/Components/ScanQR.tsx

```diff
@@ -125,6 +129,7 @@ const ScanQR: React.FC<IProps> = ({ kiosk }) => {
                     How do I get a game's share link or QR code?
                 </a>
             </div>
+            <ErrorModal showing={!!addingError} errorType={addingError} errorDescription={errorDesc!} setShowing={setAddingError}/>
```

Comment:
Suggestion: Unconditionally render `ErrorModal` here, and let the component decide whether it should actually render.